rs/crypto/internal/crypto_service_provider/src/signer/tests.rs

```diff
@@ -344,33 +340,6 @@ mod verify_ed25519 {
         assert_eq!(csp.verify(&sig, &msg, AlgorithmId::Ed25519, pk), Ok(()));
     }
 
-    /// Runs basic tests for batch verification of Ed25519 signatures for different input sizes.
-    /// More extensive tests can be found in the tests of `ic-crypto-internal-basic-sig-ed25519` crate.
-    #[test]
-    fn should_correctly_verify_batch_consistently_with_non_batched() {
```

Comment:
Ditto previous comment re test redundancy; we test this case also in `ic-crypto-ed25519` but only with smaller batch sizes.

Reply:
@randombit, so that I can merge, I'm resolving this open conversation because you approved the MR.